src/content/news/update-v1-3.md

```
@@ -0,0 +1,118 @@
+-    **Smoke Break**: HP-5 > HP-3.
+-    **Poison Cure**: HP+10 > HP+12.
+-    **Stark's Lightning Strike**: DMG 7 + HP/7 > DMG 10 + HP/8. Removes Priority-1 requirement.
+
+### Stark
+-    Base Stat:
+      - HP: 120 > 125
+-    **Offensive Stance** and **Defensive Stance**: SPD+1 > SPD+2.
+-    **Jumbo Berry Special Break**: Heal 7HP > Heal 10HP.
+-    **Fear Brought Me This Far**: Remove the 60HP conditional and make it give 3/3 on base.
+-    **Lightning Strike**: Reworked to **Last Stand**
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)